src/foam/u2/view/TableCellFormatter.js

```diff
@@ -144,11 +144,25 @@ foam.CLASS({
     {
       class: 'foam.u2.view.TableCellFormatter',
       name: 'tableCellFormatter',
-      value: function(value) {
-        this.start()
-          .style({'text-align': 'left', 'padding-right': '20px'})
-          .add('$' + (value/100).toFixed(2).replace(/(\d)(?=(\d{3})+\.)/g, '$1,'))
-        .end();
+      value: function(value, obj, axiom) {
+        if ( axiom.unitPropName ) {
```

Comment:
Since we're refining `UnitValue`, `axiom` should always be a `UnitValue`, and `unitPropName` should always be set. I think it would be better to log a warning if it's not set and just output the unformatted value in that case.